newrelic/core/environment.py

```diff
@@ -198,11 +198,13 @@ def environment_settings():
 
     env.extend(dispatcher)
 
+    return env
+
+
+def plugins():
     # Module information.
     stdlib_builtin_module_names = _get_stdlib_builtin_module_names()
 
-    plugins = []
-
     settings = global_settings()
     if settings and settings.package_reporting.enabled:
```

Comment:
Since we already check for this in the harvester we should probably remove it from here as it's not needed.